sender/http.go

```diff
@@ -211,7 +207,7 @@ func (ht *HTTP) initStats() {
 // to re-implement them.
 func (ht *HTTP) sendBytes(b []byte) error {
 	if !ht.ok {
-		return skogul.Error{Reason: "HTTP sender not in OK state", Source: "http-sender"}
+		return fmt.Errorf("HTTP sender not in OK state")
```

Comment:
The same as above. Capital letter. Maybe the error message could say something more verbose? Ex. "http sender failed to initialize"